README.md

```diff
@@ -63,6 +63,7 @@ git get <REPO> [flags]
 
 Flags:
   -b, --branch              Branch (or tag) to checkout after cloning.
+  -D, --depth               Create a shallow clone with a history truncated to the specified number of commits.
   -d, --dump                Path to a dump file listing repos to clone. Ignored when <REPO> argument is used.
```

Comment:
tbh I would switch around these short-hands, as `-D` is more commonly used for dumping/debugging... But that would be a breaking change. Up to you @grdl!